src/plugins/controls/public/options_list/components/options_list_strings.ts

```diff
@@ -126,7 +119,13 @@ export const OptionsListStrings = {
       }),
     getSortDisabledTooltip: () =>
       i18n.translate('controls.optionsList.popover.sortDisabledTooltip', {
-        defaultMessage: 'Ignore sorting when “Show only selected” is true.',
+        defaultMessage: 'Sorting is ignored when “Show only selected” is true',
+      }),
+    getDocumentCountTooltip: (documentCount: number) =>
+      i18n.translate('controls.optionsList.popover.documentCountTooltip', {
+        defaultMessage:
+          'This value appears in {documentCount} {documentCount, plural, one {document} other {documents}}',
+        values: { documentCount },
```

Comment:
@elastic/kibana-docs This text is for the tooltip that shows up when hovering on the document change.

![Dec-21-2022 16-29-34](https://user-images.githubusercontent.com/8698078/209022754-70467548-9b2a-436c-9bca-4d3d9bf64afc.gif)